### PR TITLE
Csalas navbar

### PR DESCRIPTION
**Pull request by SalasC2**

### CODE REVIEW

**Review by francisli — CHANGES_REQUESTED** [review]
This doesn't seem to match the designs in Figma- when you open the EcoFoodie project in Figma, make sure to switch to "Version 2" in the pages list to see the latest version.

It should have a light green background, and links on the header that collapse into a menu on mobile...?

Screen Shot 2020-03-23 at 9 19 29 PM
Screen Shot 2020-03-23 at 9 18 08 PM

**Comment by SalasC2** [comment]
Changed navbar to match 
Expected Navbar

On actual website
Screen Shot 2020-03-24 at 5 57 35 PM


TODO: Still need to add logic to change between sign in and sign out depending on whether a user is logged in or not.

File: views/layout.ejs [hunk 1]
```diff
@@ -6,7 +6,8 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <title><%= title %> - <%= process.env.APP_NAME %></title>
-    <link rel="stylesheet" href="/libraries/bootstrap/css/bootstrap.css" />
+      <!-- Bootstrap core CSS -->
+    <link href="stylesheets/bootstrap.css" rel="stylesheet">
```
Comment (francisli) [inline views/layout.ejs]: Did it not work to simply override certain styles in Bootstrap?

By copying the entire file and then modifying specific styles, that makes it more difficult to upgrade the Bootstrap version...?

**Comment by francisli** [comment]
Also need to override button and nav link styles to match the green...?

**Comment by francisli** [comment]
Does this collapse properly on narrow widths (I.e. phone?)

**Comment by francisli** [comment]
Ok, I'm going to merge this in and delete the branch, so pull the latest on master and create a new branch for continued work...

**Pull request merged**